support/src/figwheel/client/utils.cljs

```diff
@@ -120,7 +120,8 @@
 
 (defonce local-persistent-config
   (let [a (atom {})]
-    (when (feature? js/localStorage "setItem")
+    (when (and (html-env?)
```

Comment:
As I mentioned in the description, I don't understand why `feature?` doesn't work on its own here as it has in the past. Explicit checking for `html-env?` seems naive, since it might be provided by a polyfill/lib in non-browser environments.